### DISCUSSION TITLE
Не работает команда добавить цикл из временной таблицы на смартфоне

### DISCUSSION THREAD

**Discussion by horyachka** [totumonline/totum-ru-issues-and-discussions#612] [ОШБК!] [answered]
Добрый день.
Во временной таблице создал кнопку "Добавить цикл" в таблицу циклов. На ноутбуке кнопка добавляет цикл и сразу его открывает в окне, а на смартфоне - не добавляет (нажали > кнопка на секунду провалилась > восстановилась и пользователь остался во временной таблице).
Подскажите пожалуйста: что нужно сделать, чтобы кнопка на смартфоне заработала?
Скрипт для кнопки:

a1= : insert(table: 'table_all_cycles'; inserts: "inserts"; log: true)

a2=: set(table: 'table_in_cycle'; cycle: $#inserts[0]; field: 'h_date' = $date; log: true)
date: $#ndt

a3=: linkToTable(table: 'table_in_cycle'; cycle: $#inserts[0]; title: "*"; target: "blank")

Спасибо!

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Попробуйте изменить target: "blank"  на target: "self" тк браузел на мобильном может блокировать всплывающее окно. Уже как много времени блокировка - это стандартное поведение для браузеров.

**Reply by horyachka** [reply]
Алексей, СПАСИБО! "blank" поменял на "self" и кнопка на смартфоне заработала!!!
